include/glow/Base/TensorSerialization.h

```diff
@@ -52,6 +54,32 @@ void dumpToRawTextFile(Tensor &tensor, llvm::StringRef filename);
 /// round-trips (dump/load) but is used for human readability.
 void loadFromRawTextFile(Tensor &tensor, llvm::StringRef filename);
 
+/// Load network input tensor (same one images are loaded into) from the tensor
+/// blobs files in \p filenames. All the loaded tensors are concatanated along
+/// the batch dimension. The default loader expected the following tensor blob
+/// format:
+///   -- first line: 4 space separated values representing NCHW dimensions (W is
+///   the fastest).
+///   -- second line: Space separated float value that are loaded into \p
+///   tensor.
+/// TODO: Default tensor laoder could be extended to support various data types
+/// or image layout.
+void loadInputTensorFromFileWithType(
```

Comment:
nit: The input `tensor` is always reset, so maybe it would be cleaner/make more sense to just return a Tensor here instead of passing a pointer to a Tensor?

Reply:
I was trying to match `loadImagesAndPreprocess` which does the similar job. Please confirm you are ok with this one taking different signature.

Reply:
Hi @jfix71 , pinging you about this questions -please let me know if I should change or leave as is.

Reply:
@vuzelac-cadence I'm fine either way, if you want to leave it for consistency fine by me.

Reply:
Ok thanks, I prefer to leave as is.